### PR TITLE
V3 appveyor android build release only

### PR DESCRIPTION
**Pull request by crazyhappygame**

### CODE REVIEW

**Comment by crazyhappygame** [comment]
@drelaptop could you merge it?

File: .appveyor.yml [hunk 1]
```diff
@@ -11,7 +11,7 @@ environment:
     - build_type: windows32_cocos_new_test
     - build_type: windows32
     - build_type: android_cpp_tests
-    - build_type: android_cocos_new_test
+#    - build_type: android_cocos_new_test
```
Comment (drelaptop) [inline .appveyor.yml:14]: what's the reason to remove `android_cocos_new_test` task?

Comment (crazyhappygame) [reply .appveyor.yml:14]: To speedup appveyor builds.
android_cocos_new_test is run in travis as well.
I see that appveyor is much slower than travis. 
p.s.
All appveyor build configurations except windows32 were added by me 6 months ago :)

Comment (drelaptop) [reply .appveyor.yml:14]: that's fine :)

**Pull request merged**